.gitignore

```diff
@@ -154,4 +154,3 @@ wandb
 
 # Checkpoints, config files and temporary files created in tutorials.
 .hydra/
-*.jsonl
```

Comment:
just making sure -- is this because we want to add some jsonl files on the unit test?

Reply:
Yea